### PR TITLE
Removed astropy helpers from various files

### LINKED ISSUE #3674: Removing astropy_helpers section in CONTRIBUTING.rst
### Description

As of PR https://github.com/sunpy/sunpy/pull/3598, sunpy no longer needs `astropy_helpers`, and even it is removed from the package.
I think there should not be a section of Astropy Helpers in contribution guidelines as well.

### PR DESCRIPTION
**Pull request by abhijeetmanhas**
### Description

Fixes #3674 and more.

1) Should I remove mentions of astropy helpers in `MANIFEST.in` too? I found some licenses references in that file so I didn't.

2) `sunpy/version.py` mentions "This file is for compatibility with astropy_helpers", so do wee need it?

### CODE REVIEW

**Comment by nabobalis** [comment]
> 1. Should I remove mentions of astropy helpers in `MANIFEST.in` too? I found some licenses references in that file so I didn't.

Yeah, we shouldn't need those anymore.

> 2. `sunpy/version.py` mentions "This file is for compatibility with astropy_helpers", so do wee need it?

Can you delete it and see if everything still works?

**Comment by abhijeetmanhas** [comment]
@nabobalis removed version file. Let's see what happens!

File: MANIFEST.in [hunk 1]
```diff
@@ -23,24 +23,3 @@ recursive-include examples *
...
-include astropy_helpers/ah_bootstrap.py
-
-recursive-include astropy_helpers/astropy_helpers *.py *.pyx *.c *.h *.rst
-recursive-include astropy_helpers/astropy_helpers.egg-info *
-
-prune astropy_helpers/build
-prune astropy_helpers/astropy_helpers/tests
-
-# Globally exclude compiled files
-global-exclude *.pyc *.o
```
Comment (nabobalis) [inline MANIFEST.in]: I think we want these.

File: .pep8speaks.yml [hunk 2]
```diff
@@ -4,7 +4,6 @@ pycodestyle:
     - setup.py
     - ez_setup.py
     - ah_bootstrap.py
```
Comment (nabobalis) [inline .pep8speaks.yml]: Remove theses.

Comment (nabobalis) [reply .pep8speaks.yml]: Also if these appear anywhere else too.

Comment (abhijeetmanhas) [reply .pep8speaks.yml]: even setup.py is to be removed from everywhere?

Comment (abhijeetmanhas) [reply .pep8speaks.yml]: Doing it

Comment (nabobalis) [reply .pep8speaks.yml]: `ez_setup.py` is an old file. It is not `setup.py`.

**Comment by abhijeetmanhas** [comment]
@nabobalis it didn't failed. Sorry for force push , I thought not doing so will increase commits for trivial changes. I have made all changes you suggested. https://github.com/sunpy/sunpy/pull/3676/files can be viewed for quick review.

File: sunpy/version.py [hunk 3]
```diff
@@ -1,13 +0,0 @@
-# This file is for compatibility with astropy_helpers
```
Comment (Cadair) [inline sunpy/version.py]: We do still need this file.

Comment (nabobalis) [reply sunpy/version.py]: What for?

Comment (Cadair) [reply sunpy/version.py]: It calculates our version at runtime: https://github.com/sunpy/sunpy/blob/master/sunpy/__init__.py#L35

Comment (nabobalis) [reply sunpy/version.py]: Just at runtime? The version appears in the docs build and the pypi packages that twine checks.

Comment (Cadair) [reply sunpy/version.py]: Yeah, the docs call a different thing to get the version, and pypi uses setuptools_scm to generate it into the sdist metadata.

**Pull request merged**